docs/multi-cluster-mesh/multi-cluster-mesh.md

```diff
@@ -0,0 +1,81 @@
+# Single mesh multi cluster multiple networks with Istio operator
+
+## Objective:
+Setup three clusters (one active – two passive) in a single mesh multi network (two in the same network, one in a different network) setup with Istio operator.
+
+## Setup: 
+
+Steps:
+- Build custom cluster registry controller image:
+make docker-build docker-push IMG=[your docker hub account]/cluster-registry-controller:v0.1.1-dev.2
+- Build custom istio operator image:
+make docker-build docker-push IMG=[your docker hub account]/istio-operator:v2.11.0-dev.5 SKIP_UPDATE_DEPS=1
+
+Active setup:
+- Install cluster registry controller:
+helm install --namespace=cluster-registry --create-namespace cluster-registry-controller deploy/charts/cluster-registry --set image.repository=lac21/cluster-registry-controller --set image.tag=v0.1.1-dev.2 --set localCluster.name=laci-active --set network.name=network1
+- Install istio operator:
```

Comment:
I'd consider first installing the cluster registry on both clusters and establishing connection between the clusters (cluster cr + secret swap) first and only then turn to Istio operator and istio related steps.

We could link the CRC steps on how to do this.